its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/MultiLanguageTest.java

```diff
@@ -314,13 +315,17 @@ public class CreateGitFolder implements AutoCloseable {
 
     Path gitDir;
 
-    public CreateGitFolder(Path projectDir) throws Exception {
+    public CreateGitFolder(Path projectDir) {
       gitDir = projectDir.resolve(".git");
       deleteGitFolder();
-      // Initialize a new repository
-      Git git = Git.init().setDirectory(projectDir.toFile()).call();
-      System.out.println("Initialized empty Git repository in " + git.getRepository().getDirectory());
-      git.close();
+      try {
+        // Initialize a new repository
+        Git git = Git.init().setDirectory(projectDir.toFile()).call();
+        System.out.println("Initialized empty Git repository in " + git.getRepository().getDirectory());
+        git.close();
+      } catch (Exception ex) {
```

Comment:
I've done this for methods that forces the pointless `throws` on more than two places